shell/packages/sandstorm-accounts-ui/account-settings.html

```diff
@@ -18,92 +18,191 @@
 
 <template name="sandstormAccountSettings">
   {{setDocumentTitle}}
-  {{!-- TODO(now): better reuse here --}}
+
   {{#sandstormTopbarItem name="title" priority=5 topbar=_topbar}}
     Account Settings
   {{/sandstormTopbarItem}}
 
   <h2>My Account</h2>
-
+  <div class="identities-editor">
+  <div class="title-bar"> Linked Identities: </div>
```

Comment:
This should be `<h3>`, not `<div>`.

In general, we should avoid `<div>` any time that any other element type would make sense.